core-services/prow/02_config/openshift/cert-manager-operator-release/_pluginconfig.yaml

```
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/cert-manager-operator-release
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/cert-manager-operator-release
+  review_acts_as_lgtm: true
+plugins:
```

```suggestion
triggers:
- repos:
  - openshift/cert-manager-operator-release
  trusted_apps:
  - red-hat-konflux
plugins:
```
Just a random suggestion, with this you'd still have to check if adding this whitelists the Konflux bot for PR tests and we don't have to add the `/ok-to-test` label each time manually!

---

suggestion incorporated.